controller/src/main/java/io/pravega/controller/metrics/StreamMetrics.java

```diff
@@ -243,6 +257,42 @@ public void deleteStream(String scope, String streamName, Duration latency) {
         deleteStreamLatency.reportSuccessValue(latency.toMillis());
     }
 
+    /**
+     * This method reports the latency of ControllerEventProcessor delete Stream event processing.
+     *
+     * @param latency       Latency of the deleteStreamEvent operation.
+     */
+    public void controllerEventProcessorDeleteStreamEvent(Duration latency) {
+        controllerEventProcessorDeleteStreamLatency.reportSuccessValue(latency.toMillis());
+    }
+
+    /**
+     * This method reports the latency of ControllerEventProcessor update Stream event processing.
```

Comment:
Javadoc needs to be consistent. For `Seal` and `Truncate` you have used capital letter but `update` is not following the same

Reply:
made consistent by following small case at all places.